public/js/clients/chrome/events.js

```diff
@@ -38,8 +39,8 @@ function paused(callFrames, reason, data, hitBreakpoints, asyncStackTrace) {
     type: reason
   }, data);
 
-  actions.paused({ frame, why });
-  actions.loadedFrames(frames);
+  await actions.paused({ frame, why });
+  return await actions.loadedFrames(frames);
```

Comment:
at some point it might make sense to move `loadedFrames` into `paused` as they're doing the same thing